.gitignore

```diff
@@ -15,3 +15,6 @@
 /log/*
 !/log/.keep
 /tmp
+.stuff
```

Comment:
Actually, I maintain mockups related to all my projects in the .stuff folder which I don't push to GitHub since it's heavy in size